README.md

```diff
@@ -30,6 +30,8 @@ The attributes listed below are used in *course.json* to configure **Resources**
 
 **_resources** (object): The Resources object that contains values for **title**, **description**, **_filterButtons**, **_filterAria**, and **_resourcesItems**.  
 
+>**_isEnabled** (boolean): Turns **Resource** on and off. Acceptable values are `true` and `false`. 
```

Comment:
In all other places it's 'Resource**s**' not 'Resource' - you just need to make the word plural